content/blog/2017-11-16-ropensci-changes.md

```diff
@@ -0,0 +1,104 @@
+---
+slug: "ropensci-changes"
+title: "changes: easy Git-based version control from R"
+date: 2017-11-16
```

Comment:
please set date 2017-11-28

If you decide to add an image, instructions for how to do that are here: https://github.com/ropensci/roweb2#contributing-a-blog-post